spec/factories/users.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :administrator, class: User do
+    name { 'factory_name1' }
+    email { 'admin@factory.com' }
+    password { 'aaaaaa' }
+    admin { 'true' }
+  end
+  factory :user, class: User do
+    name { 'factory_name2' }
+    email { 'user@factory.com' }
+    password { 'bbbbbb' }
+    admin { 'false' }
```

Comment:
boolean型のカラムのデータは文字列ではなくbooleanの値で定義した方が良いです。
```suggestion
    admin { false }
```

Reply:
修正済みです。 8e85d15